osu.Game/Overlays/MusicController.cs

```diff
@@ -400,35 +400,38 @@ private void onTrackCompleted(WorkingBeatmap workingBeatmap)
                 NextTrack();
         }
 
-        private bool allowRateAdjustments;
+        private bool allowTrackAdjustments;
 
         /// <summary>
-        /// Whether mod rate adjustments are allowed to be applied.
+        /// Whether mod track adjustments are allowed to be applied.
         /// </summary>
-        public bool AllowRateAdjustments
+        public bool AllowTrackAdjustments
         {
-            get => allowRateAdjustments;
+            get => allowTrackAdjustments;
             set
             {
-                if (allowRateAdjustments == value)
+                if (allowTrackAdjustments == value)
                     return;
 
-                allowRateAdjustments = value;
+                allowTrackAdjustments = value;
                 ResetTrackAdjustments();
             }
         }
 
         /// <summary>
-        /// Resets the speed adjustments currently applied on <see cref="CurrentTrack"/> and applies the mod adjustments if <see cref="AllowRateAdjustments"/> is <c>true</c>.
+        /// Resets the adjustments currently applied on <see cref="CurrentTrack"/> and applies the mod adjustments if <see cref="AllowTrackAdjustments"/> is <c>true</c>.
         /// </summary>
         /// <remarks>
-        /// Does not reset speed adjustments applied directly to the beatmap track.
+        /// Does not reset any adjustments applied directly to the beatmap track.
         /// </remarks>
         public void ResetTrackAdjustments()
         {
-            CurrentTrack.ResetSpeedAdjustments();
+            CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Balance);
+            CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Frequency);
+            CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Tempo);
+            CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Volume);
```

Comment:
I feel like a cleaner way to write this is to add a parameterless overload of `RemoveAllAdjustments` to `DrawableTrack` which would house these 4 lines. Should I open a framework PR for that?